LongitudinalDifferenceInDifferencesPy/didDesigner.py

```diff
@@ -378,13 +385,11 @@ def printPercentImprovement(self, df):
     #---------------------------------------------------------------------------------------
     #Test 4 - Percent Improvement
     #---------------------------------------------------------------------------------------
-    kpiTreat0Post0_mean, kpiTreat0Post1_mean, kpiTreat1Post0_mean, kpiTreat1Post1_mean = 10, 12, 10, 20
-    print('pre-treatment mean KPI of comparison group: ' + str(kpiTreat0Post0_mean) + \
-        '\npost-treatment mean KPI of comparison group: ' + str(kpiTreat0Post1_mean) + \
-        '\npre-treatment mean KPI of treated group: ' + str(kpiTreat1Post0_mean) + \
-            '\npost-treatment mean KPI of treated group: ' + str(kpiTreat1Post1_mean))
+    
+    #this needs to be updated to call the function
+    kpiTreat0Post0_mean, kpiTreat0Post1_mean, kpiTreat1Post0_mean, kpiTreat1Post1_mean = 4.0, 9.0, 3.0, 13.0
     percentImprovement = ((kpiTreat1Post1_mean - (kpiTreat1Post0_mean + \
         (kpiTreat0Post1_mean - kpiTreat0Post0_mean))) / (kpiTreat1Post0_mean + \
             (kpiTreat0Post1_mean - kpiTreat0Post0_mean)))*100
-    print('percent improvement: ' + str(percentImprovement) + '%')
-    assert(percentImprovement == ((2 / 3) * 100))
+    assert(percentImprovement==obj.calculatePercentImprovement(df))
```

Comment:
Thanks for calling the function. This assert statement isn't quite doing the right thing. We should be asserting that `((2 / 3) * 100) == obj.calculatePercentImprovement(df))`

Reply:
Please add spaces between operators.

Reply:
Why is the correct assertion (2/3)*100? That isn't the correct output given the formula.

Reply:
I updated to (5/8)*100 which is the correct result